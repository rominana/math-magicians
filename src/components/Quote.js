import React from 'react';
import '../styles/Quote.css';

const Quote = () => (
  <div className="quote-section">
    <h2 className="quote-title">Mathematics may not teach us to add love or subtract hate, but it gives us hope that every problem has a solution.</h2>
    <p className="quote-p">â€” Anonymous</p>
  </div>
);

export default Quote;
