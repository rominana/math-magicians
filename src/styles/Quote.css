.quote-section {
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -100%);
  margin: 2rem;
  display: flex;
  flex-direction: column;
  color: white;
}

.quote-title {
  font-size: 2rem;
  margin-bottom: 1rem;
}

.quote-p {
  font-size: 1.5rem;
  margin: 0;
  align-self: end;
}
