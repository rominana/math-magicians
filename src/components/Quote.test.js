import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

it('Renders Quote component', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
