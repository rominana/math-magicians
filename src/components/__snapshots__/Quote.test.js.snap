// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Quote component 1`] = `
<div
  className="quote-section"
>
  <h2
    className="quote-title"
  >
    Mathematics may not teach us to add love or subtract hate, but it gives us hope that every problem has a solution.
  </h2>
  <p
    className="quote-p"
  >
    â€” Anonymous
  </p>
</div>
`;
